etc/adminhtml/system/adyen_hpp.xml

```diff
@@ -33,44 +33,55 @@
             <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
             <config_path>payment/adyen_hpp/active</config_path>
         </field>
+        <field id="title" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Title</label>
+            <comment>The title of each local payment method is set automatically.</comment>
+            <config_path>payment/adyen_hpp/title</config_path>
+        </field>
         <field id="sort_order" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="0">
             <label>Sort Order</label>
             <frontend_class>validate-number</frontend_class>
             <config_path>payment/adyen_hpp/sort_order</config_path>
         </field>
-        <group id="adyen_hpp_openinvoice_settings" translate="label" showInDefault="1" showInWebsite="1" showInStore="1" sortOrder="100">
-            <label>Klarna\RatePay\Afterpay Settings</label>
+        <group id="adyen_hpp_store_payment_details" translate="label" showInDefault="1" showInWebsite="1" showInStore="1" sortOrder="100">
+            <label>Store payment details options</label>
+            <comment>Adyen can securely store the payment details for select local payment methods of your shoppers, allowing you to accept recurring payments.</comment>
             <frontend_model>Magento\Config\Block\System\Config\Form\Fieldset</frontend_model>
-            <field id="ratepay_id" translate="label" type="text" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
-                <label>RatePAY Device Ident SId</label>
-                <tooltip>Unique RatePAY Id provided by RatePAY integration consultant</tooltip>
-                <config_path>payment/adyen_hpp/ratepay_id</config_path>
+            <field id="enable_recurring" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1"
```

Comment:
is this sortOrder correct?

Reply:
there is only one field in this group, so doesn't really matter